xbmc/cores/AudioEngine/Sinks/AESinkDARWINIOS.mm

```diff
@@ -546,7 +546,7 @@ static void EnumerateDevices(AEDeviceInfoList &list)
   device.m_displayNameExtra = "";
   // TODO screen changing on ios needs to call
   // devices changed once this is available in active
-  if (CServiceBroker::GetWinSystem()->GetCurrentScreen() > 0)
+  if (0)
```

Comment:
shouldn't you remove the if-condition entirely in this case?

Reply:
If you look at the code it was already `if (false)` before this PR.